src/sentry/static/sentry/app/views/releases/detail/overview/index.tsx

```diff
@@ -285,6 +338,20 @@ function getDropdownOptions(): DropdownOption[] {
       value: 'p50',
       label: t('Slow Transactions'),
     },
+    {
+      sort: {kind: 'desc', field: 'trend_percentage()'},
+      query: 'tpm():>0.01 trend_percentage():>0% t_test():<-6',
+      trendType: 'regression',
+      value: 'regression',
+      label: t('Trending Regressions'),
+    },
+    {
+      sort: {kind: 'asc', field: 'trend_percentage()'},
+      query: 'tpm():>0.01 trend_percentage():>0% t_test():>6',
```

Comment:
in hindsight, I should've made `t_test` confidence and let the endpoint handle whether it should be -6 or 6. I'll revisit this in a later update so we don't need `DropdownOptions.query`